### DISCUSSION TITLE
How many 🇺🇸 presidents have served more than two terms?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#764] [Galaxy Trivia] [answered]
**Bonus:** Explain why 🧐

**Comment by peterbaccaro** [comment, marked as answer]
**Only one U.S. president* has served more than two terms, and that was Franklin D. Roosevelt**. He was elected to the presidency four times, serving from 1933 until his death in 1945. However, following his death, the 22nd **Amendment to the United States Constitution was passed in 1951, limiting future presidents to a maximum of two terms in office**.

**Comment by SANKARAMDAS** [comment]
Franklin D. Roosevelt is the only American president to have served more than two terms.
4,422days  March 4, 1933 – April 12, 1945 Three full terms; died 2 months and 23 days into fourth term
